gpustat/core.py

```diff
@@ -189,8 +199,12 @@ def _repr(v, none_value='??'):
         # temperature and utilization
         reps = "%(C1)s[{entry[index]}]%(C0)s " \
             "%(CName)s{entry[name]:{gpuname_width}}%(C0)s |" \
-            "%(CTemp)s{entry[temperature.gpu]:>3}'C%(C0)s, " \
-            "%(CUtil)s{entry[utilization.gpu]:>3} %%%(C0)s"
+            "%(CTemp)s{entry[temperature.gpu]:>3}'C%(C0)s, "
```

Comment:
We should use a dedicated color for fan speed.

Reply:
Do you think yellow/bold_yellow is okay?